great_expectations/validation_operators/actions.py

```diff
@@ -132,7 +132,7 @@ class StoreAction(NamespacedValidationAction):
 
     def __init__(self,
                  data_context,
-                 target_store_name,
+                 target_store_name="local_validation_result_store",
```

Comment:
I think these target_store_name parameters should be fetched from the data_context provided so that the default can automatically change if the user updates their top-level `validations_store_name` key or `evaluation_parameter_store_name` key. So the default would look like:

```
target_store_name=None
...
if target_store_name is None:
  target_store_name = data_context.validations_store
```